requirements/edx/github.txt

```diff
@@ -58,7 +59,7 @@ git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-clie
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
 -e git+https://github.com/edx/edx-organizations.git@release-2015-08-31#egg=edx-organizations
 
-git+https://github.com/edx/edx-proctoring.git@0.8.3#egg=edx-proctoring==0.8.3
+git+https://github.com/edx/edx-proctoring.git@044a63d701e1516dae7861e8964b2666825d54e0#egg=edx-proctoring==0.7.3
```

Comment:
@chrisndodge That makes sense.  In terms of timing, I was thinking:

1) Wait for the 9/11 RC to get cut.
2) Rebase my branch of edx-proctoring and bump the version number to 0.9.x.
3) Wait for full code review of this PR and edx-proctoring.
4) Final rebase of edx-proctoring (if there are any other changes), then merge.
5) Update requirements in this PR to point to the final version of edx-proctoring, then merge.